### DISCUSSION TITLE
Daily 22/9

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#20] [General]
Ayer redacte los pasos de la instalación de CMS y subí la tarea al foro. Avancé muy poco con el contenido de CSS. Le dediqué mas tiempo al repaso de Inglés. No pude avanzar con las US.
Para hoy tengo previsto leer los contenidos de Hosting y Dominio de SGC. Continuar con el repaso de Inglés y  con la lectura de CSS.
La dificultad que veo para hoy es el poco tiempo del que dispongo para llevar adelante lo propuesto.

**Comment by rpgrca** [comment]
Yo ayer hice el primer ejercicio de Python de los opcionales para hacer de repaso de Cisco, también armé una segunda versión del pdf de requisitos y lo subí en la parte de la documentación. Para el proyecto voy a armar más historias y conversar sobre empezar a resolverlas.

Hoy hay clases de Sistemas de Contenido así que espero estar, tal vez también entre a la clase de Full-Stack del aula de Natalia para comparar el avance ya que están bastante más avanzados que nosotros. Es posible que haga el examen de inglés también.

**Comment by Brian-Almada** [comment]
Ayer seguí intentando descargar Wordpress y una vez más no lo conseguí. Hoy será otro intento.
Luego con el tiempo que me quede voy a seguir repasando Python

**Reply by AhumadaCeleste** [reply]
Brian, cual es la dificultad especifica que tienes?

**Reply by Brian-Almada** [reply]
El problema fue xampp me parece, porque yo lo tenía de antes y le había hecho modificaciones internas a las carpetas. Así que lo desinstalé y lo volví a instalar y pude, al fin, descargar Wordpress en local. Gracias por preguntar, Cele.

**Comment by AhumadaCeleste** [comment]
Ayer estudie Ingles y Tablas de HTML. Hoy realice el examen de Ingles, y comencé a diseñar una posible estética para nuestra pagina de Subastas. Avanzare un poco con HTML. Mañana Espero terminar HTML para comenzar con CSS. Veré la clase de Gestión de Contenidos que no podré ver hoy, y asistiré a la FullStack.

**Comment by rocappellano** [comment]
Seguí estudiando inglés y realizando actividades de esta misma materia, también repase teórico de Programador Fullstack

**Comment by Thiagox749** [comment]
Hoy completé la actividad de CMS, repaso de contenido y puesta en practica de HTML, CSS y JavaScript.
Mañana planeo ver las clases de esta semana de Fullstack y Gestion de contenidos; ademas de seguir repasando ingles.
